docs/building_lbann.rst

```diff
@@ -293,6 +293,8 @@ Hydrogen, and LBANN separately, by whatever means they choose.
           ${LBANN_HOME}/superbuild
 
         ninja
+        ml use ${LBANN_INSTALL_DIR}/etc/modulefiles/
+        ml load lbann-0.99.0
```

Comment:
We'll need to remember to update this every time we up the version.

Reply:
What could go wrong?